### DISCUSSION TITLE
Support for FreeRTOS

### DISCUSSION THREAD

**Discussion by PratikAher20** [nasa/cFS#930] [Ideas]
@dzbaker and @dmknutsen https://github.com/nasa/osal/issues/620#issuecomment-1303272348 I had this issue. Can you please help me to get the support for FreeRTOS-CFS.

**Comment by dzbaker** [comment]
Hi @PratikAher20. We don't provide support for FreeRTOS, but it looks like support has been developed here: https://github.com/pztrick/cfs-freertos.
